enhancements/testing/improved-platform-tests.md

```diff
@@ -0,0 +1,412 @@
+---
+title: improved-platform-tests
+authors:
+  - "@soltysh"
+reviewers:
+  - "@ardaguclu"
+  - "@bertinatto"
+  - "@deads2k"
+  - "@ingvagabund"
+approvers:
+  - "@deads2k"
+api-approvers:
+  - None
+creation-date: 2022-11-16
+last-updated: 2023-05-31
+tracking-link:
+  - https://issues.redhat.com/browse/WRKLDS-720
+replaces:
+  - "enhancements/testing/extended-platform-tests.md"
+---
+
+# Improved Platform Tests
+
+## Summary
+
+Allow maintaining and building end-to-end (e2e) tests outside of
+[origin](https://github.com/openshift/origin/) repository, and running them under
+single `openshift-tests` umbrella, by extracting the necessary test binaries
+during runtime.
+
+## Motivation
+
+The reasoning behind this change are two-folded:
+
+1. Short term goal is to speed up the [kubernetes updates](#kubernetes-updates).
```

Comment:
```suggestion
1. Short term goal is to speed up the [kubernetes updates](#kubernetes-updates) by enabling us to test the proposed rebase against the updated upstream tests instead of old versions of them"
```